skillmap/src/components/SkillGraphContainer.tsx

```diff
@@ -67,6 +73,14 @@ export class SkillGraphContainerImpl extends React.Component<SkillGraphContainer
             }
         }
 
+        if (backgroundColor === strokeColor) {
+            altTextColor = 'white';
+        }
+
+        if (maps.length > 1) {
+            backgroundAltText = `Background image for ${maps?.length} connected skillmaps`
```

Comment:
```suggestion
            backgroundAltText = lf("Background image for {0} connected skillmaps", maps?.length || 0)
```

and no lf below -- the first parameter to an lf needs to be a string with " or ', not a var~